includes/multi-currency/class-frontend-prices.php

```diff
@@ -51,6 +51,10 @@ public function __construct( Multi_Currency $multi_currency ) {
 			add_filter( 'woocommerce_coupon_get_amount', [ $this, 'get_coupon_amount' ], 50, 2 );
 			add_filter( 'woocommerce_coupon_get_minimum_amount', [ $this, 'get_coupon_min_max_amount' ], 50 );
 			add_filter( 'woocommerce_coupon_get_maximum_amount', [ $this, 'get_coupon_min_max_amount' ], 50 );
+
+			// Subscription product hooks.
+			add_filter( 'woocommerce_subscriptions_product_price', [ $this, 'get_product_price' ], 50, 2 );
+			add_filter( 'woocommerce_subscriptions_product_sign_up_fee', [ $this, 'get_product_price' ], 50, 2 );
```

Comment:
What do you think of adding these hooks to the [filters test](https://github.com/Automattic/woocommerce-payments/blob/develop/tests/unit/multi-currency/test-class-frontend-prices.php#L48)? I think the `has_filter` check should work even if Subscriptions is not installed as part of the unit tests.